src/renderer/components/confirm-dialog/confirm-dialog.tsx

```diff
@@ -124,10 +114,10 @@ export class ConfirmDialog extends React.Component<ConfirmDialogProps> {
       <Dialog
         {...dialogProps}
         className={cssNames("ConfirmDialog", className)}
-        isOpen={dialogState.isOpen}
+        isOpen={isOpen}
         onClose={this.onClose}
         close={this.close}
-        {...(dialogState.isOpen ? { "data-testid":"confirmation-dialog" } : {})}
+        {...(isOpen ? { "data-testid": "confirmation-dialog" } : {})}
```

Comment:
Why `data-testid` passed only when dialog is open?

Reply:
I questioned this myself, it is so that while the dialog is closed `findByTestId` won't find it.

Reply:
In general we should not render dialog itself, but not hiding reachability in tests. 
But this is fine for now I guess.